src/HHClientFallbackHandler.hack

```diff
@@ -132,7 +132,7 @@ class HHClientFallbackHandler extends FailureHandler {
       return false;
     }
 
-    $killswitches = ImmSet {'CI', 'TRAVIS', 'CONTINUOUS_INTEGRATION'};
+    $killswitches = vec['CI', 'TRAVIS', 'CONTINUOUS_INTEGRATION'];
```

Comment:
Why not keyset?

Reply:
Because a vec is continuous in memory and no set-like behaviors were used.
Both keyset and vec would work equally well here.

Reply:
Heh, I had the opposite stance: no vec-like behaviors are used  - it's basically a 'contains' check